extension.json

```diff
@@ -248,6 +248,34 @@
       "libPath": "dist/lib/dataElements/eventMergeId/index.js",
       "viewPath": "dataElements/eventMergeId.html"
     },
+    {
+      "displayName": "Identity Map",
+      "name": "identity-map",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "type": "object",
+        "additionalProperties": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "authenticatedState": {
+                "type": "string",
+                "enum": ["loggedOut", "authenticated", "ambiguous"]
+              },
+              "primary": {
+                "type": "boolean"
+              }
+            }
+          }
+        }
+      },
```

Comment:
Looking at `https://github.com/adobe/reactor-extension-alloy/pull/96/files#diff-1fa3fffe87cc751f1039a9f91f2cd92aR13`, it seems that the identity map data should be located at `settings.data`, but the schema doesn't look like it reflects this (there's no `data` property in the schema).

Reply:
yes, thank you, I thought I changed that